app/konnect/org-management/okta-idp.md

```diff
@@ -87,7 +87,7 @@ claims to extract that information.
 
 1. Go to the Claims tab.
 
-    You need to configure two claims: `groups` and `login_email`.
+    Configure the claim: `groups`.
```

Comment:
```suggestion
```
I was working with this stuff yesterday and `groups` is something you specify, so it's not the actual name of something in the UI.